### DISCUSSION TITLE
Download failed lifesize public

### DISCUSSION THREAD

**Discussion by JaimeXen** [aclap-dev/video-downloadhelper#747] [Help and Q&A]
### Your question

Previously, it was possible to download public links from Lifesize

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://playback.lifesize.com/#/publicvideo/1e67fc45-638c-42a8-a6f7-60fe63d0c8ad?vcpubtoken=5e8df563-4c25-475b-a05c-014d7b8cb865",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 000001547d6aa680] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://iospriv-vc.lifesize.com/30f85275-6046-4db1-9800-26e4cb15c1a1/recordings/by-date/2024/05/28/1e67fc45-638c-42a8-a6f7-60fe63d0c8ad-16-44/mainfeed_pip_1152_iosfeed/ios_45-1152-ios_private.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_e915272a-5198-4054-a37f-77ab2d86bdd3",
    "downloadable_id": "downloadable_2941028011434739"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

This site is like a number of others being reported here.  There is an HLS master manifest available that you can find in the Network Monitor.  But when I run ffprobe on it, I get 403 Forbidden.  The web site has applied a kind of stealth DRM.  Real DRM would be reported directly at the top of the browser page & Firefox isn't reporting that.  But a 403 error means you're supposed to supply some sort of authentication to get access to the file.  Of course, the web site doesn't publish that information so there's no way to download the content.  You're out of luck here.  It's not going to work.  You'll have to find other content you can download.
